docs/usage.md

```diff
@@ -57,6 +57,10 @@ work                # Directory containing the nextflow working files
 # Other nextflow hidden files, eg. history of pipeline runs and old logs.
 ```
 
+### Memory considerations
+
+The pipeline needs to handle large amounts of data, depending on the size and number of microbiomes the user has defined in the input. To handle these data the pipeline mainly uses python scripts and the python module pandas. As the data needs to be compared memory consumption is currently peaking at around 150 GB for the full size test, but can easily be higher depending on the input. Therefore, computing on high mem nodes with enough allocated memory is recommended. If the memory is still an issue one can try to reduce the chunk sizes for high memory consuming processes. The parameters are: `--proc_chunk_size` for the preprocessing of the peptides prior to the epitope prediction, `--pred_chunk_size` for the epitope prediction process and `--ds_prep_chunk_size` for the downstream analysis processes.
```

Comment:
```suggestion
The pipeline needs to handle large amounts of data, depending on the size and number of microbiomes the user has defined in the input. To handle these data the pipeline mainly uses python scripts and the python module pandas. As the data needs to be compared memory consumption is currently peaking at around 150 GB for the full-size test, but can easily be higher depending on the input. If the memory is still an issue one can try to reduce the chunk sizes for high memory consuming processes. The parameters are: `--proc_chunk_size` for the preprocessing of the peptides prior to the epitope prediction in `SPLIT_PRED_TASKS`, `--pred_chunk_size` for the epitope prediction process `PREDICT_EPITOPES` and `--ds_prep_chunk_size` for the downstream analysis processes `PREPARE_ENTITY_BINDING_RATIOS` and `PREPARE_SCORE_DISTRIBUTION`.
```

- I would suggest to add the actual process names, since this might help users to map the information to observed failed processes.
- the "computing on high mem nodes" I would not write, since 1) not each process has to be computed on a high mem node, 2) what's the definition on high mem node? Mentioning 150GB will be enough I guess. I will adjust the `base.config` file, so the users hopefully will get an better idea also based on this and then we can add further information in the future if necessary

Reply:
Thats true, thanks for the input!

Yes I wasn't quite sure if the info will be more for developers or users, but still error will post the corresponding process, so probably best to name that!